registrations/tests.py

```diff
@@ -4050,3 +4050,77 @@ def test_return_personnel_codes(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(sorted(response.data['results']), ['11111', '22222'])
+
+
+class TestTransferRegistrationsCommand(AuthenticatedAPITestCase):
+
+    def make_registration(self, operator_id):
+        reg_data = REG_DATA['hw_pre_mother']
```

Comment:
Maybe we should make a copy of this dictionary instead? So that when we modify it below, we're not changing it for future tests.